config/search-query/filter-form.ttl

```diff
@@ -0,0 +1,89 @@
+@prefix : <#>.
+@prefix dct: <http://purl.org/dc/terms/>.
+@prefix foaf: <http://xmlns.com/foaf/0.1/>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix schema: <http://schema.org/>.
+@prefix lblodSubsidie: <http://lblod.data.gift/vocabularies/subsidie/>.
+@prefix form: <http://lblod.data.gift/vocabularies/forms/>.
+@prefix sh: <http://www.w3.org/ns/shacl#>.
+@prefix displayTypes: <http://lblod.data.gift/display-types/>.
+@prefix ext: <http://mu.semte.ch/vocabularies/ext/>.
+@prefix mu: <http://mu.semte.ch/vocabularies/core/> .
+@prefix search: <http://redpencil.data.gift/vocabularies/search-queries/> .
+@prefix searchSubsidie: <http://lblod.data.gift/vocabularies/search-queries-subsidie/> .
+
+
+##########################################################
+# form
+##########################################################
+  ext:form a form:Form, form:TopLevelForm ;
+    mu:uuid "e025a601-b50b-4abd-a6de-d0c3b619795c";
+    form:includes ext:typeSubsidieF;
+    form:includes ext:typeBestuurF;
+    form:includes ext:bestuurseenheidF;
+    form:includes ext:aanvraagDatumF;
+    form:includes ext:subsidieStatusF.
+
+##########################################################
+#  property-group
+##########################################################
+ext:applicationPg a form:PropertyGroup.
+
+##########################################################
+#  Filter fields
+##########################################################
+ext:typeSubsidieF a form:Field ;
+    sh:name "Subsidie" ;
+    sh:path searchSubsidie:subsidieMeasureOffer;
+    form:displayType displayTypes:conceptSchemeSelector ;
+    search:emberQueryParameterKey "subsidietype" ;
+    form:options  """{"conceptScheme":"http://lblod.data.gift/concept-schemes/5979ff3e-1f4c-4271-ae40-e1d2000d6412"}""" ;
+    form:help "vb. Contact-en bronopsporing, Fietsinfrastructuur" ;
+    sh:order 10;
+    sh:group ext:applicationPg .
```

Comment:
The figma mockup has placeholders for the filters, which are currently not present in the frontend. Not sure if it's necessary

Reply:
https://github.com/lblod/ember-submission-form-fields/blob/master/addon/components/rdf-input-fields/concept-scheme-selector.hbs if we add a placeholder attribute here, it should be pretty easy

Reply:
I think they are not really needed in this case. We have both a label & helptext so placeholder might be overkill. Also I dont think it is supported by ember-submission-form-fields currently